app/Controller/AppController.php

```diff
@@ -671,4 +680,16 @@ public function sendJSON($data)
         $this->autoRender = false;
         return $this->response->body(json_encode($data));
     }
+
+    public function IPisBan() {
```

Comment:
Ajoute en paramètre l'ip directement

Reply:
Donc si je dois appeler la fonction ça donnerait, IPisBan($this->Util->getIP());
?

Reply:
Et j'enlève la récupération d'IP à l'intérieur de la fonction

Reply:
Ok je te fais ça